src/main/java/org/apache/commons/imaging/Imaging.java

```diff
@@ -16,9 +16,6 @@
  */
 package org.apache.commons.imaging;
 
-import static org.apache.commons.imaging.ImagingConstants.PARAM_KEY_FILENAME;
-import static org.apache.commons.imaging.ImagingConstants.PARAM_KEY_FORMAT;
```

Comment:
N.B. no generics warnings being suppressed in `Imaging` now :tada:

Reply:
Woo hoo!